### ISSUE TITLE
''The Fate of Kagrosh'' Quest in Shadowmoon Valley

### ISSUE THREAD

**Issue by Mickekk** [dalaranwow/dalaran-wow#5043]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: When I click on the corpse of Kagrosh and choose option ''Search the corpse for Kagrosh's pack'' nothing happens. I abandoned the quest and tried to do it again, but result is the same.

**Expected behaviour**: After choosing option ''Search the corpse for Kagrosh's pack'' I should receive ''Kagrosh's Pack'' in order to turn in the quest and continue with the quest chain.

**Steps to reproduce the problem**:

1. Pick up a quest from ''Scout Zagran''
2. Enter the Deathforge in Shadowmoon Valley
3. Click on the corpse of Kagrosh and choose option ''Search the corpse for Kagrosh's pack'' - nothing happens

**Include proofs for this behaviour**

http://www.wowhead.com/quest=10601/the-fate-of-kagrosh - URL to the quest with video.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

http://www.wowhead.com/item=30659/kagroshs-pack - This is the item I should get in order to turn in the quest.

**Include Screenshots from the issue if necessary**

**Comment by Lothloryen** [comment]
Hello there,

Just checked this quest and it seems to work just fine, are you sure that you don't have the item in your inventory already?

**Comment by Mickekk** [comment]
Hello, I have the quest, but I don't have the item. I tried three times so far.

**Comment by Lothloryen** [comment]
Hello again, nevermin, my bad. There is an issue which was deployed yesterday that broke several quests with mechanics similar to this one. It has been fixed already and all those quests will be working again within the next maintenance (in 1-2 hours).

Thank you for your report.

**Comment by Mickekk** [comment]
Thanks for the answer.

There are some more bugs in Shadoowmoon Valley I have experienced during questing there.

First one is when quest says that I have to kill some number of mobs and when I kill one it gives me credit for two kills.

Second one is regarding quest ''Teron Gorefiend - Lore and Legend''. The NPC ''Ancient Shadowmoon Spirit'' doesn't appear now at all, but couple of hours before I had the quest, he was where he was supposed to be . Now I can't turn in that quest at all and continue with chain.

**Comment by Lothloryen** [comment]
Try all of them after the maintenance please, if any of those persist, open a new issue for each, it will be appreciated and looked into.

Regards.
